App/Services/WeddingService.cs

```diff
@@ -47,8 +47,16 @@ public List<Guest> getTheMostInconvenientGuests(List<Guest> guests)
                     dayIndexWithMinimumBusyPeople = entry.Key;
                 }
             }
+
+            if (dayIndexWithMinimumBusyPeople == 0)
+            {
+                return new List<Guest>();
+            }
+            else
+            {
+                return guestsWorkingDaysMatrix[dayIndexWithMinimumBusyPeople];
+            }
```

Comment:
**Nit:** `else` block is not needed here since there is a return in the main if block. It's called an early return pattern. From my point of view, early returns make code cleaner and easier to read / maintain.

```suggestion
            if (dayIndexWithMinimumBusyPeople == 0)
            {
                return new List<Guest>();
            }
            
            return guestsWorkingDaysMatrix[dayIndexWithMinimumBusyPeople];
```